src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.animal.AnimalComparator;
+
+
+/**
+ * Sorts all animals in the zookeep book to the user.
```

Comment:
Think a better description could be `Sorts all animals in the zookeep book based on user specified parameter`?